lib/mobile_robot_simulator/robot_models/ackerman_robot.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright (c) 2011 Autonomous Robots and Cognitive Systems Laboratory, Universidad de Costa Rica
+# Copyright (c) 2011 Autonomous Robots and Cognitive Systems Laboratory
```

Comment:
Sometimes this reads  Autonomous Robots and Cognitive Systems Laboratory, Universidad de Costa Rica and others  Autonomous Robots and Cognitive Systems Laboratory; without the Universidad de Costa Rica. Standarize to one or the other please.

Reply:
The one that includes UCR is prefered. Thanks for the observation.